server/controllers/api/addFlight_post.js

```diff
@@ -0,0 +1,19 @@
+import models from '../../database/models/index';
+
+export default(req, res, next) => {
+  const {
+    originValue, destinationValue, dateValue, airCraftTypeValue, flightDuration, flightNumberValue,
+  } = req.body;
+  models.Flights.create({
+    flight_no: flightNumberValue,
+    origin: originValue,
+    destination: destinationValue,
+    flight_date: dateValue,
+    flight_duration: flightDuration,
+    aircraft: airCraftTypeValue,
+  }).then((result) => {
+    if (result) {
+      res.status(200).send({ succesfullyAdded: true });
```

Comment:
Here you could use a status code of 201, which means "successfully created something" (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201).  

Also, it might prove to be helpful to the frontend if the response also contains the flight_id of the created flight.  The frontend might then have enough information to be able to add the new flight to the list of flights in the UI without having to re-fetch the whole list.